src/lib/y2storage/boot_requirements_strategies/base.rb

```diff
@@ -68,18 +68,35 @@ def needed_partitions(target)
         planned_partitions
       end
 
-      # All boot errors detected in the setup, for example, when a /boot/efi partition
-      # is missing in a UEFI system
+      # All boot warnings detected in the setup, for example, when required partition is too small
       #
       # @note This method should be overloaded for derived classes.
       #
       # @see SetupError
       #
       # @return [Array<SetupError>]
-      def errors
+      def warnings
         []
       end
 
+      # All fatal boot errors detected in the setup, for example, when a / partition
+      # is missing
+      #
+      # @note This method can be overloaded for derived classes.
+      #
+      # @see SetupError
+      #
+      # @return [Array<SetupError>]
+      def errors
+        res = []
+        if root_filesystem_missing?
+          error_message = _("There is no device mounted to '/'")
```

Comment:
NP: `There is no device mounted at '/'` ??

Reply:
:+1: